app/assets/stylesheets/dropzone.css

```diff
@@ -0,0 +1,388 @@
+/*
```

Comment:
Would it be possible to add a comment detailing how this file differs from / interacts with basic.css above?

Reply:
Is this a file I should be touching for frontend or no?

Reply:
I think this is from Kate's ticket, so she might know more about where it came from